src/plugin.js

```diff
@@ -190,6 +197,7 @@ const onPlayerReady = (player, options) => {
         <h2 class="vjs-errors-headline">${this.localize(error.headline)}</h2>
           <div><b>${this.localize('Error Code')}</b>: ${(error.type || error.code)}</div>
           ${details}
+          <i><span class="vjs-errors-flashmessage">${flashMessage}</span></i>
```

Comment:
Personally, since the flash message already has style associated with it, I'd rather see that in CSS and have the `<i>` removed.